modules/sdpolicy-networking.adoc

```diff
@@ -7,6 +7,12 @@
 
 [id="custom-domains_{context}"]
 == Custom domains for applications
+
+[NOTE]
+====
+Starting with {product-title} 4.14 on, the Custom Domain Operator is deprecated, and you can manage Ingress with the Ingress Operator. The functionality is unchanged for {product-title} 4.13 and earlier versions. 
```

Comment:
I think you could remove the "on" here based on what I've seen earlier in this PR.
```suggestion
Starting with {product-title} 4.14, the Custom Domain Operator is deprecated, and you can manage Ingress with the Ingress Operator. The functionality is unchanged for {product-title} 4.13 and earlier versions. 
```